### ISSUE TITLE
minimal instructions/docstring for Trie

### ISSUE THREAD

**Issue by mikkokotila** [Esukhia/botok#72]
I've tried to look for information on the required input parameters for Trie(bosyl, profile, main_data, custom_data) but realized that it's not contained anywhere in a single place. Could you provide minimal description for each, and some indication of recommended values for some common use-cases. 

My understanding is that I can reduce overhead by loading Trie only once in my application, and then handling tokenization as I need throughout the app. As opposed to just using Text.

**Comment by 10zinten** [comment]
duplicate of #73
